src/allotropy/parsers/moldev_softmax_pro/softmax_pro_structure.py

```diff
@@ -437,28 +437,6 @@ def create(reader: CsvReader, header: PlateHeader) -> PlateRawData:
 
         return plate_raw_data
 
-    @staticmethod
```

Comment:
I see why you moved this, but moving it to the `PlateData` class isn't ideal either, as is break encapsulation.

I would suggest either:

1. Keeping 1 `RawPlateData` class, and having `create` do the different logic based on read_type
2. Having a base class `RawPlateData` that is a base class to two subclasses `SpectrumRawPlateData` and something like `EndpointAndKineticRawPlateData`, then keeping common logic (like this function) in the base class